### DISCUSSION TITLE
Remove Price Limits in OpenRCT2 like the Ride Price Manager Plugin

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#17995] [Feature requests] [answered]
Can you remove the 20 dollar cap on prices for rides and stalls. When you go to change the price of a Stall Item, there is no requirement to charge over 20 dollars, and most people would leave most items around 1 dollar (or pound etc) but I think that you should be allowed to charge whatever you want. If you use the plus button, it would take a long time to increase it, so most people would click on the price and enter it manually. I don't think it would harm anyone to give players the choice to enter a higher price value for all items than 20 dollars. It has been proven by Marcel and others that some rides peeps are willing to spend more than 25 dollars for, but the game by default will not allow it, and some parks (such as a theoretical Gigapark) might benefit from a higher allowed park entrance fee, if the scenario creator gives peeps more starting cash.

**Comment by spacek531** [comment]
Is this even possible? I thought sale prices were stored as uint8_t fixed-point to the tens so the max sale price would be 25.50

**Reply by 733737** [reply]
are you sure about this? $25.50

**Comment by duncanspumpkin** [comment]
It would also be pointless as no one would buy them.

**Reply by 733737** [reply]
umbrellas would be bought at any price peeps could afford, and there is a give peeps 1000 credits cheat

**Comment by 733737** [comment]
I would be fine with increasing the cap on umbrellas to 25.50

**Comment by ageekhere** [comment]
Personally I think the max price for umbrellas should be lowered to about $5.00

**Comment by 733737** [comment]
the point of OpenRCT2 is not to make the game harder, or change for the sake of change, the point is to improve the game using modern code

**Comment by 733737** [comment]
https://github.com/OpenRCT2/OpenRCT2/blob/develop/src/openrct2/ride/ShopItem.cpp

I don't see a maximum price listed here, these are the prices peeps will pay, if I want to submit a PR, which Header or code file should I look at?

**Reply by ocalhoun6** [reply]
It will be a bit more complicated for umbrellas because there is no maximum price peeps are willing to pay for an umbrella when it's raining. If it's raining, they'll buy it no matter what (if they can afford it).

Having peeps think that umbrellas are too expensive during rain will require a little bit of new game logic, not just modifying a single number in a header file.

**Reply by 733737** [reply]
I have absolutely no interest in this feature. The developers have said before they don't want to rebalance game, and neither do I

**Reply by ageekhere** [reply]
Another idea could be if it is raining and peeps are forced to buy an umbrella at say $20.00 their happiness drops significantly as a penalty for price gouging.

At the moment there is no downside to charging max price for umbrellas, this idea would create a downside allowing players to decided if they want to take the risk of price gouging umbrellas.

**Reply by 733737** [reply]
Or you could just stop derailing the conversation

**Comment by 733737** [comment, marked as answer]
I have been told by Gymnasiast that this is an unpopular change, and I assume new elements would also disapprove, so I am no longer pursuing this change in the codebase
